script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -666,7 +666,7 @@ But having you help out all day seems a little rough,
 -- Page 48, Offset 24382.
 -- さんざん夜に出歩いておいて何だけど、それは世間一般においてデート、と言うものじゃないんだろうか？
 I know we've already spent a good deal of time walking around at night together,
- but isn't what she's describing now awfully close to what people might consider a date?
+ but isn't what she's describing now is awfully close to what people might consider a date?
```

Comment:
Yes, doesn't it?

Reply:
No, as in the `is` is a mistake.
> **Isn't** what she's describing now ~~is~~ awfully close to what people might consider a date?

Adding the `is` is incorrect because you already have the `isn't` on the front.
> What she's describing now is awfully close to what people might consider a date.

would be correct, but doesn't work with the question.

Reply:
Oh my bad, that one really fooled me